app/src/main/res/values-ja/localized.xml

```diff
@@ -1,2 +1,424 @@
+    <string name="mat_skill_gold_rider">騎の秘石</string>
+    <string name="mat_skill_gold_caster">術の秘石</string>
+    <string name="mat_skill_gold_asssassin">殺の秘石</string>
+    <string name="mat_skill_gold_berserker">狂の秘石</string>
+
+    <string name="mat_skill_red_saber">剣の魔石</string>
+    <string name="mat_skill_red_archer">弓の魔石</string>
+    <string name="mat_skill_red_lancer">槍の魔石</string>
+    <string name="mat_skill_red_rider">騎の魔石</string>
+    <string name="mat_skill_red_caster">術の魔石</string>
+    <string name="mat_skill_red_asssassin">殺の魔石</string>
+    <string name="mat_skill_red_berserker">狂の魔石</string>
+
+    <string name="mat_skill_blue_saber">剣の輝石</string>
+    <string name="mat_skill_blue_archer">弓の輝石</string>
+    <string name="mat_skill_blue_lancer">槍の輝石</string>
+    <string name="mat_skill_blue_rider">騎の輝石</string>
+    <string name="mat_skill_blue_caster">術の輝石</string>
+    <string name="mat_skill_blue_asssassin">殺の輝石</string>
+    <string name="mat_skill_blue_berserker">狂の輝石</string>
+
+    <string name="p_shuffle_cards">コマンドシャッフル</string>
+    <string name="p_shuffle_cards_wave">ウェーブ</string>
+    <string name="p_shuffle_cards_when">タイミング</string>
+    <string name="p_shuffle_cards_when_none">使用しない</string>
+    <string name="p_shuffle_cards_when_no_effective">WEAKカードがない場合</string>
+    <string name="p_shuffle_cards_when_no_np_matching">宝具使用サーヴァントのカードがない場合</string>
+
+    <string name="p_game_area_mode">ゲームエリア設定</string>
+    <string name="p_game_area_default">デフォルト</string>
+    <string name="p_game_area_duo">Surface Duo</string>
+    <string name="p_game_area_custom">カスタム</string>
+    <string name="p_game_area_custom_left">左側</string>
+    <string name="p_game_area_custom_top">上部</string>
+    <string name="p_game_area_custom_right">右側</string>
+    <string name="p_game_area_custom_bottom">下部</string>
+
+    <string name="p_welcome">FGAへようこそ!</string>
+    <string name="p_welcome_description">この画面では、最も重要なセットアップ手順を説明します。</string>
+    <string name="p_skill_confirmation_onboarding_description">FGOの「スキル使用確認」はONですかOFFですか？\n\nFGOの初期設定では、スキルを使用するたびに「決定」を押すよう求められます。\n\nこの設定は後で変更することができます。</string>
+    <string name="p_on">ON</string>
+    <string name="p_off">OFF</string>
+    <string name="p_battery_optimization">バッテリー使用の最適化</string>
+    <string name="p_battery_optimization_description">FGAはバッテリー使用最適化設定を「制限なし」にする必要があります。そうしないと、時間が経つとランダムに動作しなくなります。\n\nボタンを押した後、アプリフィルターを「最近開いたアプリ」から「すべてのアプリ」に切り替え、FGAを「制限なし」に設定してください。</string>
+    <string name="p_battery_optimization_action">バッテリー使用最適化が無効</string>
+    <string name="p_battery_optimization_dontkillmyapp">また、お使いのデバイス固有の最適化を無効にする必要があるかもしれません。ガイドはdontkillmyapp.comでご覧いただけます。</string>
+    <string name="p_youtube_guide">Youtube ガイド</string>
+    <string name="p_youtube_guide_description">Youtube ガイドを見ることをお勧めします。このビデオで設定方法を教えてくれます。</string>
+    <string name="p_youtube_guide_action">ガイドを開く</string>
+    <string name="donate">Donate</string>
```

Comment:
Although this can also be translated directly, it is a notation that frequently appears in foreign-made applications, and it would be more helpful to leave it as it is.